anchor-reference-server/src/main/java/org/stellar/anchor/reference/AnchorReferenceServer.java

```diff
@@ -79,6 +106,7 @@ static String getProperty(String name, String defaultValue) {
 
   public static void main(String[] args) {
     start(
+        new HashMap<>(),
```

Comment:
This is a little weird since the `envMap` is also created at [L33](https://github.com/stellar/java-stellar-anchor-sdk/pull/909/files#diff-d9049317c2f758da352a24d0c7a82c9b80f3ea48154b90fba4387364f9920c01R33).

Reply:
IntelliJ was warning me about passing in a `null` here. Let me see if there's another way to do this.

Reply:
This is no big deal because we are deprecating it. Not worth the time fixing it unless you can't resist the urge. 😄

Reply:
Actually, it's not warning me anymore. It was probably doing so because I had not initialized the empty map on L33 previously.